bucket/scantailor-advanced.json

```diff
@@ -0,0 +1,34 @@
+{
+    "description": "A fork of ScanTailor that merges the features of the ScanTailor Featured and ScanTailor Enhanced versions, brings new ones and fixes.",
```

Comment:
This description do not state what scintailor is in general.

Something like `Interactive post-processing tool for scanned pages`